pcl_catkin/CMakeLists.txt

```diff
@@ -27,19 +27,24 @@ SET(PCL_CXX_FLAGS "-Wall -Wextra -Wno-unknown-pragmas -fno-strict-aliasing -Wno-
 SET(PCL_CXX_FLAGS "${PCL_CXX_FLAGS} -std=c++11")
 
 ExternalProject_Add(pcl_src
-  GIT_REPOSITORY  https://github.com/ethz-asl/pcl
-  GIT_TAG pcl_1.8.0_devel
+  GIT_REPOSITORY  https://github.com/PointCloudLibrary/pcl
+  GIT_TAG pcl-1.9.1
   UPDATE_COMMAND ""
   CMAKE_ARGS -DCMAKE_INSTALL_PREFIX:PATH=${CATKIN_DEVEL_PREFIX}
              -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
              -DCMAKE_CXX_FLAGS:STRING=${PCL_CXX_FLAGS}
+             -DPCL_SHARED_LIBS=TRUE
              "-DCMAKE_CXX_FLAGS_${UC_BUILD_TYPE}=${CMAKE_CXX_FLAGS_${UC_BUILD_TYPE}} ${PCL_CXX_FLAGS}"
 )
 
 cs_add_library(${PROJECT_NAME} src/dependency_tracker.cc)
 add_dependencies(${PROJECT_NAME} pcl_src)
 
-cs_install()
+install(DIRECTORY ${CATKIN_DEVEL_PREFIX}/lib/
+        DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+      FILES_MATCHING PATTERN "libpcl*")
+
+# cs_install()
```

Comment:
not sure anymore why I had to do this :-S

Reply:
I can try with the default cs_install again, I was fiddling around with this for it to build and then was to lazy to retry with this once it build ;). Will try this weekend though.

Reply:
yup, seems to work now, the issues were due to the submodules not depending on pcl_catkin but system pcl.